source/Superluminal-RESTfulAPI/RESTfulAPIClient.class.st

```diff
@@ -302,6 +301,31 @@ RESTfulAPIClient >> putAt: aLocation configuredBy: aRequestBuildingBlock withSuc
 			]
 ]
 
+{ #category : 'invoking - covenience' }
+RESTfulAPIClient >> query: anEntity at: aLocation accepting: aMediaType withSuccessfulResponseDo: aBlock [
+
+  ^ self
+      queryAt: aLocation
+      configuredBy: [ :request |
+          request headers setAcceptTo: aMediaType.
```

Comment:
For both commands to be applied, I think this should be:
(request headers setAcceptTo: aMediaType) + (request body contents: anEntity)